components/camel-salesforce/pom.xml

```
@@ -34,9 +34,10 @@
 
     <properties>
         <camel.osgi.export>
-            org.apache.camel*;version=${camel.version}
+            org.apache.camel*;version=${camel.version},
```

```suggestion
            org.apache.camel*;version=${camel.version}
```
Remove useless trailing comma